entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/AgentImpl.java

```diff
@@ -410,138 +410,131 @@ public void setRdaGr2PlaceOfDeath(Map<String, List<String>> rdaGr2PlaceOfDeath)
-	if (tmpDateOfTermination == null) {
-	    tmpDateOfTermination = fillTmpMap(Arrays.asList(getDateOfTermination()));
+		return tmpDateOfDeath;
 	}
 
-	return tmpDateOfTermination;
-    }
-
-    @Override
-    @Deprecated
-    public void setRdaGr2Gender(Map<String, List<String>> rdaGr2Gender) {
+	@Override
+	@Deprecated
+	public void setRdaGr2DateOfEstablishment(Map<String, List<String>> rdaGr2DateOfEstablishment) {
+		// TODO Auto-generated method stub
+		
+	}
 
-    }
+	@Override
+	@Deprecated
+	public Map<String, List<String>> getRdaGr2DateOfEstablishment() {
+		//if not available
+		if (getDateOfEstablishment() == null)
+			return null;
+		//if not transformed
+		if (tmpDateOfEstablishment == null) {
+			tmpDateOfEstablishment = fillTmpMap(Arrays.asList(getDateOfEstablishment()));
+		}
+
+		return tmpDateOfEstablishment;
+	}
 
-    @Override
-    public Map<String, List<String>> getRdaGr2Gender() {
-	// if not available
-	if (getGender() == null)
-	    return null;
-	// if not transformed
-	if (tmpGender == null)
-	    tmpGender = fillTmpMap(Arrays.asList(getGender()));
+	@Override
+	@Deprecated
+	public void setRdaGr2DateOfTermination(Map<String, List<String>> rdaGr2DateOfTermination) {
+		// TODO Auto-generated method stub
+		
+	}
 
-	return tmpGender;
-    }
+	@Override
+	@Deprecated
+	public Map<String, List<String>> getRdaGr2DateOfTermination() {
+		//if not available
+		if (getDateOfTermination() == null)
+			return null;
+		//if not transformed
+		if (tmpDateOfTermination == null) {
+			tmpDateOfTermination = fillTmpMap(Arrays.asList(getDateOfTermination()));
+		}
+
+		return tmpDateOfTermination;
+	}
 
-    @Override
-    @Deprecated
-    public void setRdaGr2ProfessionOrOccupation(Map<String, List<String>> rdaGr2ProfessionOrOccupation) {
-	// TODO Auto-generated method stub
+	@Override
+	@Deprecated
+	public void setRdaGr2Gender(Map<String, List<String>> rdaGr2Gender) {
 
-    }
+	}
 
-    @Override
-    public Map<String, List<String>> getRdaGr2ProfessionOrOccupation() {
-	return getProfessionOrOccupation();
-    }
+	@Override
+	public Map<String, List<String>> getRdaGr2Gender() {
+		//if not available
+		if (getGender() == null)
+			return null;
+		//if not transformed
+		if (tmpGender == null) 
+			tmpGender = fillTmpMap(Arrays.asList(getGender()));
 
-    @Override
-    @Deprecated
-    public void setRdaGr2BiographicalInformation(Map<String, List<String>> rdaGr2BiographicalInformation) {
-	// TODO Auto-generated method stub
+		return tmpGender;
+	}
 
-    }
+	@Override
+	@Deprecated
+	public void setRdaGr2ProfessionOrOccupation(Map<String, List<String>> rdaGr2ProfessionOrOccupation) {
+		// TODO Auto-generated method stub
+		
+	}
 
-    @Override
-    public Map<String, List<String>> getRdaGr2BiographicalInformation() {
-	return getBiographicalInformation();
-    }
+	@Override
+	public Map<String, List<String>> getRdaGr2ProfessionOrOccupation() {
+		return getProfessionOrOccupation();
+	}
 
-    @Override
```

Comment:
please revert back this code, and use EntityTypes.Agent.getEntityType() instead of the hardcoded value

Reply:
Done.